pkg/api/goprobe/server/server.go

```diff
@@ -24,16 +23,10 @@ type Server struct {
 	*server.DefaultServer
 }
 
-// SetDBPath sets the path to the database directory
-func (server *Server) SetDBPath(path string) *Server {
-	server.dbPath = path
-	return server
-}
-
 // New creates a new goprobe API server
-func New(addr string, captureManager *capture.Manager, configMonitor *config.Monitor, opts ...server.Option) *Server {
+func New(addr, dbPath string, captureManager *capture.Manager, configMonitor *config.Monitor, opts ...server.Option) *Server {
```

Comment:
Made the path a required argument for the API as it is not optional and cannot be guessed properly (just because we fall back here to the default doesn't mean this has been done elsewhere in the same way).